environment.yml

```diff
@@ -15,6 +15,8 @@ dependencies:
   - pyarrow
   - python
   - pytest
+  - pytest-rst
```

Comment:
I have a slight preference not to add these to the environment - it's intentional not to require the tableauhyperapi